docs/usage/run.md

```diff
@@ -90,15 +101,6 @@ Each of the following arguments are optional and do not need to be provided.
 > 
 > ***Example:*** `--help`
 
----  
```

Comment:
@samarth8392  Is there any particular reason why `--dry-run` is removed from CLI? May be I confused you with my comments on Friday. I meant to remove dryrun from GUI, but keep it for now at CLI ... also will be replaced with `--stub` when we transition to Nextflow.

Reply:
Dry run is still available (just moved to the `--runmode dryrun` flag suggested by @slsevilla)